.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
So for future reference, what probably happened here is that you added the node_modules folder to git ignore after you had already added its contents to the repo.  In the case this happens in the future, do a git rm to remove the files from the repository.